frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -222,6 +222,31 @@ export const CurrentVersionHeader: React.SFC<CurrentVersionProps> = ({ cv }) =>
   );
 };
 
+const ChannelHeader: React.FC<{}> = () => {
+  return (
+    <Popover
+      headerContent={<>Channel</>}
+      bodyContent={
+        <div>
```

Comment:
Nit:  use `<>` since you don't need an HTML element.

Reply:
ok, fixed